### PR TITLE
add rs /fetch action

### PR DESCRIPTION
**Pull request by chenweiyj**

### CODE REVIEW

File: lib/rs.js [hunk 1]
```diff
@@ -109,6 +109,12 @@ exports.copy = function (src, dest, callback) {
   this._rsAction(pathname, callback);
 };
 
+exports.fetch = function (url, key, callback) {
+  var encodedURL = utility.base64encode(url);
+  var pathname = '/fetch/' + encodedURL + '/to/' + this.encodeEntryURI(key);
```
Comment (fengmk2) [inline lib/rs.js]: Can you show me this qn api document link?

Comment (chenweiyj) [inline lib/rs.js]: The link is 

http://developer.qiniu.com/docs/v6/api/reference/rs/fetch.html

**Comment by fengmk2** [comment]
+1

**Comment by vingo** [comment]
http://img.xiumi.us/xmi/ua/g96T/i/ddf38a7b953327c4026fb2fe43cf75f2-sz_122543.png?x-oss-process=style/xm
above url can not work on fetch  method but others ok.  wired.

**Pull request merged**